src/client/datascience/jupyter/jupyterNotebook.ts

```diff
@@ -13,23 +9,10 @@ import cloneDeep = require('lodash/cloneDeep');
 // https://www.npmjs.com/package/@jupyterlab/services
 
 export class JupyterNotebookBase implements INotebook {
```

Comment:
Jupyter Notebook has just 2 properties (connection & session).
woo hooo...
I think INotebook can still stay, its no longer a wrapper, but an object used to hold two properties.

Reply:
Makes sense to me. Still need something to hold the connection.